### PR TITLE
AVR-DELAY ADDED

### PR DESCRIPTION
**Pull request by Mshivam2409**
Added `avr-delay` for blink code.

### CODE REVIEW

**Comment by ghost** [comment]
Is this done yet? If not please convert it to draft, so it cannot be merged prematurely.

**Comment by Mshivam2409** [comment]
This is done. I will merge it after watchdog is completed so you can do the blink part. This will break the `cargo build` command and `cargo test` command.

File: src/lib.rs
Comment (ghost) [inline src/lib.rs]: Could we remove test module as it relies on `std` which we're not using?

**Review by ghost — CHANGES_REQUESTED** [review]
This is looking good. I left a few comments that should be addressed before merging.

**Pull request merged**
